airbyte-webapp/package.json

```diff
@@ -69,6 +69,7 @@
     "react-paginate": "^8.1.3",
     "react-query": "^3.39.1",
     "react-reflex": "^4.0.9",
+    "react-resize-detector": "^8.0.3",
```

Comment:
Love that we're adding this. And we should likely also use this for the new table cell size detection for the tooltip, once this is merged. cc @krishnaglick